__tests__/components/__snapshots__/Form.spec.js.snap

```diff
@@ -41,11 +41,9 @@ exports[`Form component should match the stored snapshot 1`] = `
               value="Bulbasaur"
             />
             <div />
-            <div />
```

Comment:
interesting, was it unbalanced before ?

Reply:
@cyplo This is the JSX equivalent of `<div></div>` which I believe was created by the default help text renderer.

Reply:
ah yes, misread the tag, thought it was `</div>` :D fair enough :)